web-registry/src/components/organisms/CreditBatches.tsx

```diff
@@ -40,13 +40,13 @@ interface HeadCell {
 }
 
 const headCells: HeadCell[] = [
+  { id: 'start_date', numeric: true, label: 'date created' },
   { id: 'issuer', numeric: false, label: 'issuer' },
   { id: 'batch_denom', numeric: false, label: 'batch denom' },
   { id: 'class_id', numeric: false, label: 'credit class' },
-  { id: 'tradable_supply', numeric: true, label: 'total amount tradable', wrap: true },
+  { id: 'tradable_supply', numeric: true, label: 'total amount active', wrap: true },
```

Comment:
```suggestion
  { id: 'tradable_supply', numeric: true, label: 'total amount tradable', wrap: true },
```
This was intentional, as "active" is not really accurate here. I know the mocks are different.